### PR TITLE
Login/logout

### PR DESCRIPTION
**Pull request by KnittedSox**
Login with existing user should work. Still some error handling to be done and logout functionality not working.

### CODE REVIEW

File: minitwit/src/main/java/WebApplication.java [hunk 1]
```diff
@@ -43,11 +46,27 @@ public static void main(String[] args) {
         get("/hello", (req, res) -> "Hello");
         get(URLS.PUBLIC_TIMELINE, WebApplication.servePublicTimelinePage);
         get(URLS.REGISTER, WebApplication.serveRegisterPage);
+        get(URLS.LOGIN, WebApplication.serveLoginPage);
+        post(URLS.LOGOUT, handleLogoutRequest);
```
Comment (pizzaluc) [inline minitwit/src/main/java/WebApplication.java:50]: Any reason this is not `WebApplication.handleLogoutRequest`

**Review by pizzaluc — COMMENTED** [review]
A few binaries that need to be .gitignored

File: minitwit/src/main/java/WebApplication.java (hunk 1 above)
Comment (KnittedSox) [reply minitwit/src/main/java/WebApplication.java:50]: No, just an oversight on my part.

**Review by pizzaluc — APPROVED** [review]
Really great stuff!

**Comment by Herover** [comment]
Nice! I think it would be optimal if you can get rid of the .iml files but I don't think its very important.

**Pull request merged**